### PR TITLE
Renamings 2

### PR DESCRIPTION
**Pull request by webmaster128**
This removes the dependency from Statistics on Cache, making it possible to test components individually.

### CODE REVIEW

File: clcache.py [hunk 1]
```diff
@@ -294,6 +294,9 @@ def __init__(self, cacheDirectory=None):
         timeoutMs = int(os.environ.get('CLCACHE_OBJECT_CACHE_TIMEOUT_MS', 10 * 1000))
         self.lock = CacheLock(lockName, timeoutMs)
 
+    def openStatistics(self):
+        return Statistics(os.path.join(self.dir, "stats.txt"))
```
Comment (frerich) [inline clcache.py]: About being a bit more funky (may I say 'pythonic'?) here and define a property which returns a context manager? Like:

```python
@property
def statistics(self):
    return Statistics(os.path.join(self.dir, "stats.txt"))
```

This allows writing

```python
with cache.statistics as stats:
    stats.doThis()
```

Comment (webmaster128) [inline clcache.py]: I just implemented that in the last commit. But I did not know this property thing – will change.

**Comment by webmaster128** [comment]
Check. The property was not necessary because I changed Statistics such that I __enter__ and __exit__ the same object (just like cache.lock).

**Comment by frerich** [comment]
Merged, thanks!

**Pull request merged**